media-tv/tvbrowser-bin/tvbrowser-bin-4.2.7.ebuild

```diff
@@ -18,7 +20,8 @@ RDEPEND="|| ( dev-java/openjdk:${JAVA_SLOT}
 	dev-java/openjdk-bin:${JAVA_SLOT}
 	dev-java/openjdk-jre-bin:${JAVA_SLOT} )"
 
-S="${WORKDIR}"/"${MY_PN}-${PV}"
+RDEPEND="|| ( virtual/jdk:${JAVA_SLOT}
+	virtual/jre:${JAVA_SLOT} )"
```

Comment:
I doesn't make sense to have `|| (virtual/jdk virtual/jre )` because `virtual/jre` depends on something like `|| (virtual/jdk dev-java/jre )`, so it's already included.

I'll change this